opensquat.py

```diff
@@ -267,7 +269,7 @@ def main(self, keywords_file, confidence_level, domains_file):
     print(logo)
     print("\t\t\t"+__VERSION__+"\n")
 
-    args = parser()
+    args = get_args()
```

Comment:
```py
args = arg_parser()
```
?

Reply:
> Rename parser to arg_parser module and def parser() to def get_args() because of import problems (parser is built-in module https://docs.python.org/3.7/library/parser.html)

I had problems with `from parser import *` and `parser()` function was imported from built-in module.